packages/py-api/py_api/controllers/hackathon/participants_controller.py

```diff
@@ -95,144 +85,63 @@ def delete_participant_from_team(cls, deleted_participant: Dict[str, Any]) -> Tu
         else:
             return {"message": "The participant is not in the specified team"}, 404
 
-        # ! You're not checking success state here!!!
-        TeamFunctionality.update_team_query_using_dump(team.model_dump())
+        updated_team = TeamFunctionality.update_team_query_using_dump(
```

Comment:
Method is called `delete_participant_from_team` which is a bit confusing. Rename it to `remove_participant_from_team`.

Reply:
I believe `delete` implies that the participant document will also be deleted.